### PR TITLE
Sourcing system health status from a Rust-computed metric

### PR DESCRIPTION
**Pull request by jakrawcz-rdx**
This addresses https://whimsical.com/babylon-work-tracker-Am2s7hSa7xHd5yCxgub67a@2bsEvpTYSt1HjKXmGhM6amGA7oxHAmtGuKd and https://whimsical.com/babylon-work-tracker-Am2s7hSa7xHd5yCxgub67a@2bsEvpTYSt1HjFd8GWUvsx2i17Luct1or6A :

image

### CODE REVIEW

File: core/src/main/java/com/radixdlt/api/system/genesis/PreGenesisHealthInfoService.java [hunk 1]
```diff
@@ -79,9 +77,4 @@ public PreGenesisHealthInfoService() {}
   public NodeStatus nodeStatus() {
     return NodeStatus.BOOTING_PRE_GENESIS;
   }
-
-  @Override
-  public EventProcessor<ScheduledStatsCollecting> updateStats() {
```
Comment (LukasGasior1) [inline core/src/main/java/com/radixdlt/api/system/genesis/PreGenesisHealthInfoService.java:84]: can we now also remove the event class (`ScheduledStatsCollecting`)?

File: core/src/main/java/com/radixdlt/api/system/health/NodeStatus.java [hunk 2]
```diff
@@ -65,10 +65,35 @@
...
-  BOOTING_AT_GENESIS,
-  SYNCING,
-  UP,
-  STALLED,
-  OUT_OF_SYNC
+  BOOTING_PRE_GENESIS(
+      """
+          The Node is still executing predefined Genesis transactions."""),
+  UP(
+      """
```
Comment (LukasGasior1) [inline core/src/main/java/com/radixdlt/api/system/health/NodeStatus.java:72]: Nice descriptions :+1:

**Review by LukasGasior1 — APPROVED** [review]
There are some merge conflicts, but LGTM

File: core/src/main/java/com/radixdlt/api/system/genesis/PreGenesisHealthInfoService.java (hunk 1 above)
Comment (jakrawcz-rdx) [reply core/src/main/java/com/radixdlt/api/system/genesis/PreGenesisHealthInfoService.java:84]: yup, I removed it, and here is how GH interpreted it:
image
 :D

**Pull request merged**